gsmphys/sfc_sice.f

```diff
@@ -371,6 +372,11 @@ subroutine sfc_sice                                               &
           tem     = 1.0 / rho(i)
           hflx(i) = hflx(i) * tem * cpinv
           evap(i) = evap(i) * tem * hvapi
+          if (evap(i) .lt. -maxevap(i)) then
+              chh(i) = -maxevap(i)/evap(i)*chh(i)
+              hflx(i) = -maxevap(i)/evap(i)*hflx(i)
```

Comment:
Checking order of operations is supposed to be `(maxevap/evap) * _var_` and you want to negate it